.npmrc

```diff
@@ -0,0 +1,2 @@
+lockfile-version=3
```

Comment:
Should this files also go into the `.gitattributes` to keep it out of packaging? Or should we package it?

Reply:
ack. will do so.

Reply:
done.